### PR TITLE
chore: setup Argos Visual Testing

### PR DESCRIPTION
**Pull request by gregberge**
## :bookmark_tabs: Summary

As discussed with @sidharthv96, I setup Argos Visual Testing. At some point you should be able to replace Applitools without any limitation of usage.

## :straight_ruler: Design Decisions

I kept Applitools and `cy.matchImageSnapshot`, you should be able to remove it once Argos fits your need!

### :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://mermaid.js.org/community/contributing.html)
- [x] :computer: have added necessary unit/e2e tests.
- [x] :notebook: have added documentation. Make sure [`MERMAID_RELEASE_VERSION`](https://mermaid.js.org/community/contributing.html#update-documentation) is used for all new features.
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

**Comment by gregberge** [comment]
@sidharthv96 @knsv could you give a look?

File: cypress.config.ts [hunk 1]
```diff
@@ -21,6 +23,12 @@ export default eyesPlugin(
         // copy any needed variables from process.env to config.env
         config.env.useAppli = process.env.USE_APPLI ? true : false;
 
+        // Argos
+        registerArgosTask(on, config, {
+          uploadToArgos: !!process.env.CI,
+          token: 'fc3a35cf5200db928d65b2047861582d9444032b',
```
Comment (nirname) [inline cypress.config.ts:29]: Should not it be in the environment as well?

Comment (gregberge) [reply cypress.config.ts:29]: Of course but I can't do it for you!

Comment (sidharthv96) [reply cypress.config.ts:29]: I can set it in the env. Please add me to the team in Argos so I can regenerate it.

Comment (gregberge) [reply cypress.config.ts]: It's not possible to regenerate it yet. 😅

Comment (nirname) [reply cypress.config.ts]: We can clean the history with force push

Comment (gregberge) [reply cypress.config.ts]: Yeah it's not very sensible 😉

**Comment by gregberge** [comment]
@sidharthv96 how can I help?

**Pull request merged**